src/network/networkd-util.c

```diff
@@ -155,10 +155,6 @@ int network_config_section_new(const char *filename, unsigned line, NetworkConfi
         return 0;
 }
 
-void network_config_section_free(NetworkConfigSection *cs) {
-        free(cs);
-}
```

Comment:
i wonder if we have similar cases elsewhere…

Reply:
Not many I think. Anyway, the compiler should mostly optimize this away when doing lto. So it's mostly a matter of saving a few lines of sources.